src/harness/util.py

```diff
@@ -285,3 +288,61 @@ def convertRequestToRequestWithCpiSignature(private_key, cpi_id,
   request['cpiSignatureData']['protectedHeader'] = jwt_message[0]
   request['cpiSignatureData']['encodedCpiSignedData'] = jwt_message[1]
   request['cpiSignatureData']['digitalSignature'] = jwt_message[2]
+     
+def computePropagationAntennaModel(request):
+    reliability_level = request['reliabilityLevel']
+    if reliability_level not in [-1, 0.05, 0.95]:
+        response = 400
+        return response
+    tx = request['cbsd']
+    if ('fss' in request) and ('ppa' in request):
+        response = 400
+        return response
+    elif 'ppa' in request:
+        rx ={}
+        rx['height'] = 1.5
+        isfss = False
+        coordinates = [];
+        ppa = request['ppa']
+
+        ARCSEC = 2
+        ppa_points = ComputePPAProtectionPoints(ppa['geometry']['coordinates'], ARCSEC)
+        if len(ppa_points) == 1:
+            rx['latitude'] = ppa_points[0][0]
+            rx['longitude']= ppa_points[0][1]
+        else:
+            response = 400
+            return response
+        
+        region_val = nlcd_driver.RegionNlcdVote([[rx['latitude'], rx['longitude']]])
+
+    elif 'fss' in request:
+        isfss = True
+        rx = request['fss']
+    else:
+        response = 400
+        return response
+    
+# ITM pathloss (if receiver type is FSS) or the hybrid model pathloss (if receiver type is PPA) and corresponding antenna gains.
+# The test specification notes that the SAS UUT shall use default values for w1 and w2 in the ITM model.
+    result = {}
+    
+    if isfss:
+        path_loss = wf_itm.CalcItmPropagationLoss(tx['latitude'], tx['longitude'], tx['height'], rx['latitude'], rx['longitude'], rx['height'], reliability=reliability_level, freq_mhz=3625.)
```

Comment:
The line is long, and difficult to read, can you break some of the parameters and put them on next line ?
Same for other lines below

(for info it is recommended in the code style guideline to use around 80 columns, or a bit more but not too much)